protocols/imap/src/test/java/org/apache/james/imap/main/DefaultImapDecoderFactoryTest.java

```diff
@@ -0,0 +1,39 @@
+package org.apache.james.imap.main;
```

Comment:
THis file misses the compulsory ASF V2 license

Reply:
Added the ASF license notice to the source files. Please feel free to review and let me know if there are any concerns. Thank you! !